src/include/miopen/solver.hpp

```diff
@@ -2515,6 +2541,7 @@ struct GemmWrw1x1_stride1 : GemmWrwBase
     {
         return GetWorkspaceSize(ctx, ctx.conv_problem);
     }
+    bool MayNeedWorkspace() const { return true; }
```

Comment:
Wouldn't it be better to put MayNeedWorkspace() in the base class (GemmWrwBase)?